static/src/lang/en_GB/server.json

```diff
@@ -376,6 +376,10 @@
 			"Shamiradio",
 			"Shamiradio now playing banner"
 		],
+		"shamiradio2": [
```

Comment:
Please run `./scripts/migrate_lang.js` to replicate this in the other language packs.